packages/core/src/operator/operatorRuntime.ts

```diff
@@ -453,8 +453,10 @@ async function listenForChallengesCallback(challengeNumber: bigint, challenge: C
                 cachedLogger(`Comparison between PublicNode and Challenger was successful.`);
             })
             .catch(error => {
-                cachedLogger(`Error on CND check for challenge ${Number(challenge.assertionId)}.`);
+                cachedLogger(`Error on CDN check for challenge ${Number(challenge.assertionId)}.`);
                 cachedLogger(`${error.message}.`);
+                // Throw an error here to propagate from this catch block.
+                throw error;
```

Comment:
Why is this now throwing ?

Reply:
This has been removed, I mistakenly misunderstood it was needed to propogate errors up the stack.